<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea_about">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>565</width>
        <height>564</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_aboutVCS">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="title">
          <string>VCS vX.X.X</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>14</number>
          </property>
          <item>
           <widget class="QLabel" name="label_newVersionNotice">
            <property name="font">
             <font>
              <italic>false</italic>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A newer version of VCS is available. See &lt;a href=&quot;https://www.github.com/leikareipa/vcs/releases/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;the changelog&lt;/span&gt;&lt;/a&gt; on GitHub for more details.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>A third-party capture utility for certain Datapath capture cards. Neither associated with nor endorsed by Datapath. Copyright (c) 2017-2019 Tarpeeksi Hyvae Soft.

VCS interfaces with the Qt framework, copyright (c) The Qt Company; the RGBEasy API, copyright (c) Datapath Ltd; and the OpenCV library, copyright (c) OpenCV team. Find out more in the VCS readme.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_captureDeviceInfo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Your capture device: Unknown</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_inputChannels">
              <property name="font">
               <font/>
              </property>
              <property name="text">
               <string>Input channels:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_inputChannelsString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_inputResolution">
              <property name="text">
               <string>Input resolution, minimum:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_inputMinResolutionString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_25">
            <item>
             <widget class="QLabel" name="label_inputMaxResolution">
              <property name="text">
               <string>Input resolution, maximum:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_inputMaxResolutionString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_firmware">
              <property name="font">
               <font/>
              </property>
              <property name="text">
               <string>Firmware version:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_firmwareString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_driver">
              <property name="font">
               <font/>
              </property>
              <property name="text">
               <string>Driver version:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_driverString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_supportsDirectDMA">
              <property name="text">
               <string>Supports direct DMA:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_supportsDirectDMAString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_supportsDeinterlace">
              <property name="text">
               <string>Supports deinterlacing:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_supportsDeinterlaceString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLabel" name="label_supportsYUV">
              <property name="text">
               <string>Supports YUV encoding:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_supportsYUVString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLabel" name="label_supportsVGACapture">
              <property name="text">
               <string>Supports VGA capture:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_supportsVGACaptureString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QLabel" name="label_supportsDVICapture">
              <property name="text">
               <string>Supports DVI capture:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_supportsDVICaptureString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <widget class="QLabel" name="label_supportsComponentCapture">
              <property name="text">
               <string>Supports component capture:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_supportsComponentCaptureString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_23">
            <item>
             <widget class="QLabel" name="label_supportsCompositeCapture">
              <property name="text">
               <string>Supports composite capture:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_supportsCompositeCaptureString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_24">
            <item>
             <widget class="QLabel" name="label_supportsSVideoCapture">
              <property name="text">
               <string>Supports S-Video capture:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_supportsSVideoCaptureString">
              <property name="text">
               <string>Unknown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
